help/md/dnCDBDP.md

```diff
@@ -1,9 +1,82 @@
 ## name
 dnCDBDP
 ## title
+Character Dependent Birth Death Process
 ## description
+This function simulates a tree under a character-dependent birth-death process.
 ## details
+This function is a flexible simulator that can be used for several phylogenetic models. Specifically,
+dnCDBDP allows for character-dependent birrth-death processes meaning that the speciation/extinction rates
+depend on the character state of the taxon itself.
+ Examples of such models are outlined below:
+
+Multiple State-dependent Speciation Extinction (MuSSE)
+This model uses a state-dependent birth-death process to simulate a tree with only anagentic state changes.
+Using dnCDBDP to implement MuSSE, a vector of speciation rates for each state can be passed to the lambda argument.
+Due to being a vector of rates, dnCDBDP will anly allow anagenetic state changes (along branches), with the
+length of the vector corresponding to the number of states.
+
+Cladogenetic State-dependent Speciation Extinction (ClaSSE)
+This model allows for cladogenetic state changes (at nodes, ie. state change inducing speciation)  during the birth-death process. 
+To implement this, a cladogenetic event map must be passed to the lamda argument which will be a matrix specifiying rates of state changes 
+at nodes which will be a seperate matrix for state changes along branches which is specified in the Q argument. See example for implementation.
```

Comment:
There are too many "which"s in this sentence; I can't figure out what it's trying to say. There are also typos: "lamda" should read "lambda" (line 21), "seperate" should read "separate" (line 22).